oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/ClusterNodeInfo.java

```diff
@@ -229,6 +234,12 @@ static RecoverLockState fromString(String state) {
     /** OAK-3399 : max number of times we're doing a 1sec retry loop just before declaring lease failure **/
     private static final int MAX_RETRY_SLEEPS_BEFORE_LEASE_FAILURE = 5;
 
+    /** OAK-10622 : seconds to prevent reuse of clusterId if it crashed and thus required a recover */
+    private static final int DEFAULT_REUSE_AFTER_RECOVERY_SECS = SystemPropertySupplier.create("oak.documentMK.reuseDelayAfterRecoverySecs", -1)
+            .loggingTo(LOG).validateWith(value -> value >= -1)
+            .formatSetMessage((name, value) -> String.format("Reuse delay after recovery set to (secs): %ss (using system property %s)", name, value)).get();
+    public static final int DEFAULT_REUSE_AFTER_RECOVERY_MILLIS = 1000 * DEFAULT_REUSE_AFTER_RECOVERY_SECS;
```

Comment:
Scope could be reduced to `default` since it is only used in `ClusterNodeInfoTest`.

Reply:
+1, will reduce it

Reply:
done at https://github.com/apache/jackrabbit-oak/pull/1278/commits/697668209d86295004cde06ccaf3bd9c8585fdd3